tests/functional/u/unspecified_encoding_py38.py

```diff
@@ -162,3 +162,10 @@ class IOArgs:
 
 # Test for crash reported in https://github.com/pylint-dev/pylint/issues/6414
 open('foo', mode=2)  # [bad-open-mode, unspecified-encoding]
+
+# Infer kwargs
+KWARGS = {"mode": "rb"}
+open(FILENAME, **KWARGS)
```

Comment:
```suggestion
open(FILENAME, **KWARGS)  # [unspecified-encoding]
```
This should raise, right ?

Reply:
Ha, right. Could we add the same use cases than for the base check then ?

Reply:
Do you mean adding test cases similar to existing ones but for inferred kwargs (e.g. with no encoding or "encoding": None)

Reply:
Yes

Reply:
Yes let's add a changelog entry. It's a false positive fix and we try to release them early in the maintenance branch.